### PR TITLE
fix(jenkins) return multiple scms when jenkins has multiple scms and remoteUrl calls

### PR DESCRIPTION
**Pull request by kevinawoo**
same as https://github.com/spinnaker/igor/pull/273, but for ~`release-1.7.x`~ `release-1.8.x`

### CODE REVIEW

**Comment by lwander** [comment]
Will merge w/ #273

File: igor-web/src/main/groovy/com/netflix/spinnaker/igor/polling/CommonPollingMonitor.java [hunk 1]
```diff
@@ -126,7 +126,7 @@ public void pollSingle(PollContext ctx) {
                     );
                     sendEvents = false;
                 } else {
-                    log.warn(
+                    log.error(
```
Comment (tomaslin) [inline igor-web/src/main/groovy/com/netflix/spinnaker/igor/polling/CommonPollingMonitor.java]: this shouldn't be part of this PR

**Comment by tomaslin** [comment]
please add tests

**Comment by kevinawoo** [comment]
tests added 💯 ✅

**Comment by tomaslin** [comment]
lgtm
